thanos/README.md

```diff
@@ -2,29 +2,69 @@
 title: Thanos
 ---
 
-# Information
+Learn more about the **Thanos** Plugin. Use it to enable extended metrics retention and querying across Prometheus servers and Greenhouse clusters. 
+
+The main terminologies used in this document can be found in [core-concepts](https://cloudoperators.github.io/greenhouse/docs/getting-started/core-concepts).
+
+## Overview
+
+Thanos is a set of components that can be used to extend the storage and retrieval of metrics in Prometheus. It allows you to store metrics in a remote object store and query them across multiple Prometheus servers and Greenhouse clusters. This Plugin is intended to provide a set of pre-configured Thanos components that enable a proven composition. At the core, a set of Thanos components is installed that adds long-term storage capability to a single **kube-monitoring** Plugin and makes both current and historical data available again via one Thanos Query component.
+
+![Thanos Architecture](./img/Thanos-setup-example.png)
+
+The **Thanos Sidecar** is a component that is deployed as a container together with a Prometheus instance. This allows Thanos to optionally upload metrics to the object store and Thanos Query to access Prometheus data via a common, efficient StoreAPI.
+
+The **Thanos Compact** component applies the Prometheus 2.0 Storage Engine compaction process to block data uploaded to the object store. The Compactor is also responsible for applying the configured retention and downsampling of the data. 
```

Comment:
```suggestion
The **Thanos Compact** component applies the Prometheus 2.0 Storage Engine compaction process to data uploaded to the object store. The Compactor is also responsible for applying the configured retention and downsampling of the data. 
```

I find block data a little misleading, as it reads like the verb "to block" data.